### PR TITLE
add HiPs overlay

### PR DESCRIPTION
**Pull request by naheedsa**

### CODE REVIEW

File: lib/toggle_overlay.jpg
Comment (knro) [inline lib/toggle_overlay.jpg:1]: This is barely visible as DSS overlay. Bring the slider up

Comment (naheedsa) [reply lib/toggle_overlay.jpg:1]: i took screenshots from the Video attached in Stellarmate FB Page

Comment (knro) [reply lib/toggle_overlay.jpg:1]: In that video, I change the slider.

Comment (naheedsa) [reply lib/toggle_overlay.jpg:1]: also, this image is to only show the toggle. The effect of the DSS is in "increase_hips_opacity.jpg"

**Pull request merged**